scripts/tests/scripts_regression_tests.py

```diff
@@ -375,7 +375,7 @@ def test_a_createnewcase(self):
         self.assertTrue(os.path.exists(testdir))
         self.assertTrue(os.path.exists(os.path.join(testdir, "case.setup")))
 
-        run_cmd_assert_result(self, "./case.setup", from_dir=testdir)
+        run_cmd_assert_result(self, "./case.setup --debug", from_dir=testdir)
```

Comment:
this --debug should be removed.

Reply:
Thanks @jedwards4b, just remove the --debug.